app/views/employers/plan_years/_publish_modal.html.erb

```diff
@@ -3,17 +3,37 @@
+        <h4 class="modal-title"><%= plan_year.no_documents_uploaded?? "Document Upload Required to Proceed" : "Confirm Publish Plan Year" %></h4>
       </div>
       <div class="modal-body">
         <p>
-          As submitted, this application is ineligible for coverage under the <%= site_short_name %> exchange for reasons identified below. Click Cancel to go back and correct inaccurate information. If the information provided is accurate, you may click Publish to proceed. However, if you choose to proceed and the eligibility determination is deemed correct by <%= site_short_name %> officials, this organization may not again apply for coverage for 90 days, or until the reason for ineligibility changes, whichever is later.
+
+        <% if plan_year.no_documents_uploaded? %>
+     Before you can complete your application, you will need to submit proof of your eligibility as a Massachusetts small business to shop through the Health Connector. You will need to submit this proof before you are able to publish your plan year. <%= link_to "View our list of documents that can be submitted as proof.", "https://www.mahealthconnector.org/business/business-resource-center/employer-verification-checklist" %> 
+          <br>
+          <ul>
+            <li>
+              To upload documentation of your eligibility, go to the Documents option on the left and follow instructions to upload the documentation of your eligibility.
+            </li>
+            <li>
+              Once you have uploaded your <%= link_to 'Documents', employers_employer_profile_path(@employer_profile.id, :tab=>'documents') %> you can return to this page and publish your plan year. 
+            </li>
+          </ul>
+      
+     <% else %>        
+       As submitted, this application is ineligible for coverage under the <%= site_short_name %> exchange for reasons identified below. Click Cancel to go back and correct inaccurate information. If the information provided is accurate, you may click Publish to proceed. However, if you choose to proceed and the eligibility determination is deemed correct by <%= site_short_name %> officials, this organization may not again apply for coverage for 90 days, or until the reason for ineligibility changes, whichever is later.
+       <p><ul class="application-warnings"></ul></p>
+          <% end %>          
         </p>
-        <ul class="application-warnings"></ul>
+      
       </div>
       <div class="modal-footer">
-        <%= link_to 'Force Publish', employers_employer_profile_plan_year_force_publish_path(@employer_profile,plan_year), method: :post, :class => 'btn btn-danger' %>
-        <% error_page = plan_year.application_eligibility_warnings[:primary_office_location].present? ? edit_employers_employer_profile_path(@employer_profile, plan_year_id: plan_year.id) : edit_employers_employer_profile_plan_year_path(@employer_profile,plan_year, publish: true) %>
+        <% if plan_year.no_documents_uploaded? %>
```

Comment:
this is the only problem

We want to show go back for both options.

Just move the <% if %> to surround the Force Publish link_to. No <% else %> is necessary.